IndexProactiveControls.md

```diff
@@ -0,0 +1,166 @@
+# Table of contents
+
+<!-- Table of contents generated with markdown-toc https://ecotrust-canada.github.io/markdown-toc/ -->
+
+- [Objective](#objective)
+- [Not adressed CS](#not-adressed-cs)
+- [1. Define Security Requirements](#1-define-security-requirements)
+- [2. Leverage Security Frameworks and Libraries](#2-leverage-security-frameworks-and-libraries)
+- [3. Secure Database Access](#3-secure-database-access)
+- [4. Encode and Escape Data](#4-encode-and-escape-data)
+- [5. Validate All Inputs](#5-validate-all-inputs)
+- [6. Implement Digital Identity](#6-implement-digital-identity)
+- [7. Enforce Access Controls](#7-enforce-access-controls)
+- [8. Protect Data Everywhere](#8-protect-data-everywhere)
+- [9. Implement Security Logging and Monitoring](#9-implement-security-logging-and-monitoring)
+- [10. Handle All Errors and Exceptions](#10-handle-all-errors-and-exceptions)
+
+# Objective
+
+This index have for objective to indicate to help an [OWASP Proactive Controls](https://www.owasp.org/index.php/OWASP_Proactive_Controls) user to clearly identify which cheat sheets are useful for each section during his usage of the OWASP Proactive Controls.
+
+# Not adressed CS
```

Comment:
For the CSs that does not fit to top 10 OPC list, we use the same approach than for the ASVS, we do not mention them in this index. 

Do not worry they are referenced in the [alphabetical index](https://github.com/OWASP/CheatSheetSeries/blob/master/Index.md) (auto-generated by a script). 

The OPC index provide only useful CSs for a OPC user. 


The section *Not adressed CS* can then be removed.